demos/common/python/openvino/model_zoo/model_api/adapters/openvino_adapter.py

```diff
@@ -35,20 +34,21 @@ def create_core():
 
     log.info('OpenVINO Inference Engine')
     log.info('\tbuild: {}'.format(get_version()))
-    return IECore()
+    return Core()
 
 
 class OpenvinoAdapter(ModelAdapter):
     """
     Works with OpenVINO model
     """
 
-    def __init__(self, ie, model_path, weights_path=None, device='CPU', plugin_config=None, max_num_requests=1):
-        self.ie = ie
+    def __init__(self, core, model_path, weights_path=None, model_parameters = {}, device='CPU', plugin_config=None, max_num_requests=0):
+        self.core = core
         self.model_path = model_path
         self.device = device
         self.plugin_config = plugin_config
         self.max_num_requests = max_num_requests
+        self.model_parameters = model_parameters
```

Comment:
Why model_parameters instead of model_layouts? Are we going to modify other parameters in the model?

Reply:
We may want to provide additional information to adapter in future. So I think `model_parameters` should be dictionary